config-example.yaml

```diff
@@ -203,10 +203,18 @@ log:
   format: text
   level: info
 
-# Path to a file containing ACL policies.
-# ACLs can be defined as YAML or HUJSON.
-# https://tailscale.com/kb/1018/acls/
-acl_policy_path: ""
+## Policy
+# headscale supports Tailscale's ACL policies.
+# Please have a look to their KB to better
+# understand the concepts: https://tailscale.com/kb/1018/acls/
+policy:
+  # The mode can be "file" or "database" that defines
+  # where the ACL policies are stored and read from.
+  mode: file
+  # If the mode is set to "file", the
+  # path to a file containing ACL policies.
+  # The file can be in YAML or HuJSON format.
```

Comment:
I note the changelog mentioned that support for ACL policies in YAML format was dropped as part of this PR. Does this comment in `config-example.yaml` need updating to make it clear that only HuJSON is supported for file based now, or is the changelog incorrect?